### PR TITLE
Add initial draft and link in README

### PR DESCRIPTION
**Pull request by hillaryj**
Adds a draft of documentation on how to skeletonize / add Lineage to a repository.

## 🗣 Description

The initial sections have been tested out, there's a note on where the instructions need additional fleshing-out.

Also adds a line into the main README.

## 🧪 Testing

Pre-commit hooks passed.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Add-Lineage.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Adding Lineage to a non-skeletonized repository #
+
+## General overview ##
+
+1. In the target repo, add the skeleton as a remote
```
Comment (jsf9k) [inline Add-Lineage.md:5]: I would go ahead and mention the full `git` commands for this step and the next one.

File: Add-Lineage.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+2. Pull with `--allow-unrelated-histories`
+3. Fix all the inevitable conflicts (this takes the longest)
+4. Fix problems that may arise from `pre-commit` being added
```
Comment (jsf9k) [inline Add-Lineage.md:8]: You should probably add a step before this that mentions running `pre-commit install`.

File: Add-Lineage.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+This merge process will almost certainly fail, resulting in merge conflicts.
+The next step is to fix those conflicts and add the files once the fixes are
+in place.
+
+```console
+# Determine which files need fixes
+git status
+
+# After fixing them, add each file and then commit to complete the merge
```
Comment (jsf9k) [inline Add-Lineage.md:52]: I would make this comment more explicit, so folks don't add a single file and then commit.  Something like, "After fixing the merge conflicts in a file, `git add` it.  When all conflicts have been fixed and added, `git commit` to complete the merge."

File: Add-Lineage.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+git add <filename>
+...
+git commit --message="Merge skeleton history"
```
Comment (jsf9k) [inline Add-Lineage.md:55]: Can't we just use the default commit message here that `git` produces for the merge commit?  That message mentions the branches being merged from and to, so it is slightly more informative.

File: Add-Lineage.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+# Adding Lineage to a non-skeletonized repository #
```
Comment (jsf9k) [inline Add-Lineage.md:1]: These instructions cover a lot more than just adding Lineage.  They describe how to add a skeleton to an existing repository.  I suggest changing the title and the file name accordingly.

Also, [cisagov/development-guide](https://github.com/cisagov/development-guide) may be a better home for this document.

File: Add-Lineage.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+- `pre-commit run --all-files` - runs against all files to check the
+pre-commit hooks
+- isort and `.isort.cfg` - will have to deconflict packages (remove the
+known-first and known-third and let it populate them) and manually add the
+package name as known-first-party
+- modify the `.coveragerc` to point to the src package
+- Will need to add the appropriate secrets so they're available to the Actions
+workflow, e.g. `secrets.COVERALLS_REPO_TOKEN` for the repo badge
+  - Once the commit/repo is up, add a token from
```
Comment (jsf9k) [inline Add-Lineage.md:70]: Not all repositories need this.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have some suggestions for improvement.  In particular, I think this document covers a lot more than just adding Lineage to an existing repo; hence, I think it would be better if it lived in [cisagov/development-guide](https://github.com/cisagov/development-guide).

**Comment by hillaryj** [comment]
I'm closing this PR in favor of moving the documentation to `development-guide` as suggested. I've incorporated the feedback given here - thanks!

**Pull request closed without merging**